composer.json

```diff
@@ -15,10 +15,12 @@
     ],
     "minimum-stability": "dev",
     "require-dev": {
-        "behat/mink-selenium2-driver": "~1.1",
-        "behat/mink-extension": "~2.0",
-        "symfony/dependency-injection": "2.7.11",
-        "behat/behat": "^3.1",
-        "behat/mink-goutte-driver": "^1.2"
+        "pantheon-systems/pantheon-wordpress-upstream-tests": "dev-tagging-upstreamonly#9f241607fca96294153deaae4f8545b12"
```

Comment:
Switch back to @dev-master after https://github.com/pantheon-systems/pantheon-wordpress-upstream-tests/pull/25 is merged.